drawBot/context/baseContext.py

```diff
@@ -537,7 +537,34 @@ def transform(self, transformMatrix, center=(0, 0)):
         aT.setTransformStruct_(transformMatrix[:])
         self._path.transformUsingAffineTransform_(aT)
 
-    # boolean operations
+    def fit(self, bounds):
+        """
+        Fit and center the path inside the given bounds.
+        """
+        if self._path.isEmpty():
+            return
+        (x, y), (w, h) = self._path.bounds()
+        b_x1, b_y1, b_x2, b_y2 = bounds
+        b_w = b_x2 - b_x1
+        b_h = b_y2 - b_y1
+        if w / h >= b_w / b_h:
+            scale = b_w / w
+        else:
+            scale = b_h / h
+        self.scale(scale, scale)
+        self.center(bounds)
+
+    def center(self, bounds):
```

Comment:
I propose the following instead of center:
```python
    def align(self, point, horizontalAlignment=None, verticalAlignment=None):
        ...horizontalAlignment can be None, "center", "left" or "right"
        ...verticalAlignment can be None, "center", "top" or "bottom"
        ...`point` is an (x, y) tuple, to be used as the alignment target.
        ...for example bez.align((100, 200), "center", "bottom") will align the 
           bounding box of the shape to be centered horizontally at 100,
           and the bottom with 200. An alignment value of None will make
           the path not move in that direction.
```

Reply:
Nice, that's more versatile! Will fix.